semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -302,6 +347,62 @@ static boolean isValidURI(String uri) {
     return true;
   }
 
+  /**
+   * @param event Object encapsulating span data
+   * @return check if the event is for a sql backend
+   */
+  public static boolean isSqlBackend(Event event) {
+    if (event.getEventName() != null
+        && event.getEventName().startsWith(JDBC_EVENT_PREFIX)
+        && SpanAttributeUtils.containsAttributeKey(event, SQL_URL)) {
+      return true;
+    }
+    return isSqlTypeBackendForOtelFormat(event);
+  }
+
+  /**
+   * @param event Object encapsulating span data
+   * @return check if this span is for a mongo backend
+   */
+  public static boolean isMongoBackend(Event event) {
```

Comment:
I hope other than elasticSearch/cassandra method, rest of the changes are just re-ordering?

Reply:
yup. grouped the methods doing similar things together.